spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/config/FilterFromConfig.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.conformance.config
+
+import org.apache.log4j.{LogManager, Logger}
+import za.co.absa.enceladus.dao.rest.JsonSerializer
+import za.co.absa.enceladus.model.dataFrameFilter.DataFrameFilter
+import za.co.absa.enceladus.utils.config.ConfigReader
+
+import scala.util.{Failure, Success, Try}
+
+/**
+  * This is a helper object to allow configuration of Mapping tables filters before the UI is reasdy for that.
+  * Until then, the filters can be set via configuration.
+  */
+object FilterFromConfig {
+  private val configReader = new ConfigReader()
```

Comment:
I consider this a bit problematically testable (you have to put everything in the test's `application.conf`). If you wanted to test multiple cases, it would become increasingly harder to do.

Therefore I suggest to follow the suite of the `ConfigReader` and similar, e.g.
```scala
case class FilterFromConfig (config: Config = ConfigFactory.load()) {
  private val configReader = new ConfigReader(config)
}
```

This way, not a lot would change from the caller's perspective wanting to use it, but supplying test-configs, it would be much simpler.

Reply:
I really hope this to be a temporary solution. And that we get rid of this class/object completely. From that POV I found it good enough. Furthermore, as there can be as many configuration lines as needed for the tests, effectively independent, I though again this is "good enough". 😉

Reply:
It is good enough. I just wanted to suggest an improvement.

> as there can be as many configuration lines as needed for the tests, effectively independent
what do you mean by that?

Reply:
> 
> what do you mean by that? The fact that you can keep using different keys and thus separating different unit test cases?

Yup, exactly.

Reply:
Yes, that will hold for unit tests. Had there been some integration tests that would want to supply different config values while the keys would have to stay the same, the problem would become apparent. Which is something that we are not doing at the moment, though.

As I said, **it is good enough**. I just wanted to suggest an improvement. You do not have to follow it 😄